onmt/inputters/text_utils.py

```diff
@@ -232,12 +232,21 @@ def textbatch_to_tensor(vocabs, batch):
     for i, ex in enumerate(batch):
         if isinstance(ex, bytes):
             ex = ex.decode("utf-8")
-        toks = ex.strip("\n").split()
+        if is_train:
+            toks = ex
+        else:
+            toks = ex.strip("\n").split()
```

Comment:
This looks strange. What's the rationale?

Reply:
In training mode examples are already tokenized.